### ISSUE TITLE
[BUG] The string activity-needed-calculation-plural is unnecessarily complex

### ISSUE THREAD

**Issue by eemeli** [common-voice/common-voice#3907]
**Describe the bug**
Right now, this string breaks Pontoon in Welsh, as lifting the selectors to the message's top level results in 4^6 = 1296 variants. It's also unnecessarily complex, given that its input values are and have always been hard-coded:
https://github.com/common-voice/common-voice/blob/d19ad06c6b470b06af0fd2eefe3f0e84cca09235/web/src/components/pages/dashboard/goals/custom-goal-steps.tsx#L197-L204

**To Reproduce**
1. Go to https://pontoon.mozilla.org/ru/common-voice/web/locales/en/messages.ftl/?string=213590
2. Observe that an excessive number of edit boxes are presented.
3. Go to https://pontoon.mozilla.org/cy/common-voice/web/locales/en/messages.ftl/?string=213590
4. Observe that the browser freezes, and eventually asks the user to stop the script.

**Expected behavior**
The message should not presume that the fixed inputs are actually variable. Here's a valid alternative that's much easier to translate:
```fluent
activity-needed-calculation-plural =
  10,000 hours is achievable in just over 6 months if 1,000 people record 45 clips a day.
```

**Screenshots**
<img width="442" alt="Screenshot 2023-01-16 at 11 38 47" src="https://user-images.githubusercontent.com/617000/212657384-87bc4450-cacf-4532-9c0b-aed803d96e16.png">

**Additional context**
mozilla/pontoon#2703

It should not count as a feature that this particular string is one we tend to use to stress-test Fluent, as it's the most complex one in all of Mozilla's localizations.

**Comment by eemeli** [comment]
This string is now getting special treatment in Pontoon so that it at least doesn't break the UI for localisers.

**Comment by eemeli** [comment]
Just to let you know, under mozilla/pontoon#3538 we're changing how Pontoon serialises message files (in the next month or two).

If this message isn't fixed (e.g. by merging #3538), then the Arabic translation of this string will explode into a bit over 2000 lines. See here: https://gist.github.com/eemeli/e8ca98bd78ef8c92781afe5b9092f968

**Comment by HarikalarKutusu** [comment]
Just to be clear:

Common Voice cannot work without this sentence, right?
Simply dividing the sentence into parts, or just removing it as a whole would break everything, right?
Or would it be misleading to just give a sample statement like given above "10,000 hours is achievable in just over 6 months if 1,000 people record 45 clips a day."?
Or is it there as it is stated in the constitution?

**Comment by HarikalarKutusu** [comment]
@eemeli, I'm well aware of the issue and the solution you provided long ago. Thanks for all of your efforts.

For me, this is not-a-simple-bug, this kills the page. This one breaks some languages, it is hard to debug (I had to to give support to some RTL/Arabic script language) and it is hard to imagine why the team keeps the wording there.
If this were my software, it would take me a mere 10 minutes to get rid of it.

This is an open-source software where you point to a problem and give a solution and nobody cares to look at them. It has been 2 years!
(sorry)

**Comment by HarikalarKutusu** [comment]
Wow, even the calculation is wrong and it depends on the average sentence length (thus recording length). It is even misleading because no language can accomplish that performance.

Average recording length in CV is a bit less than 5 secs (assume 5 sec).
And this is the goal part, there is no 45 recs/day goal. We have 5, 20, 50, 100, 150.

1000 voices * 45 recs/day * 5 sec/rec => 2.6 hours/day
10000 / 2,6 => You would actually need 3845 days, more than 10 years

Actually 10k hours is also a figure from the past. And your goal depends on the complexity of the language, application, model architecture, target accuracy, and how you train (finetune a pretrained one or from scratch). By finetuning a multi-lingual Whisper you can get acceptable performance with 100 hours of data if original Whisper-multilingual already has enough data from that language. A better ultimate goal can be 1000h for example.

A better wording would perhaps be (also dividing into two to solve the issue):

> 100 people recording 100 sentences in one day can generate 13 hours of audio.
> Keeping this performance, 1000 hours can be reached in 3 months.

**Comment by HarikalarKutusu** [comment]
@eemeli I posted an alternative PR which would possibly bump the issue for being solved I hope.
#4811
